app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -783,15 +790,18 @@ private void closeTab(int index) {
           updateTabSwitcherIcon();
         })
         .show();
-    openDefaultTab();
-    updateTabSwitcherIcon();
+    if (webViewList.size() == 0) {
+      openDefaultTab();
+    }
+    else {
+      updateTabSwitcherIcon();
+    }
   }
 
   private void openDefaultTab() {
     new Handler().postDelayed(() -> {
-      if (webViewList.size() == 0) {
         newTab(HOME_URL);
-      }
+        hideTabSwitcher();  //Fix for Issue #1095
```

Comment:
Don't link issue numbers to code. This will unnecessarily blot the code base.

Reply:
Sure. I will take care the next time.